docs/source/amp.rst

```diff
@@ -25,12 +25,9 @@ However, :class:`torch.autocast` and :class:`torch.GradScaler` are modular, and
 As shown in the CPU example section of :class:`torch.autocast`, "automatic mixed precision training/inference" on CPU with
 datatype of ``torch.bfloat16`` only uses :class:`torch.autocast`.
 
-For CUDA and CPU, APIs are also provided separately:
```

Comment:
Replace mention of torch.cpu.amp.GradScaler and cuda.amp.GradScaler with just amp.Scaler in line 22

Reply:
Updated.